### ISSUE TITLE
Expose additional options to `r.tableCreate`

### ISSUE THREAD

**Issue by AndrewBarba** [neumino/thinky#489]
This is a great article from the Compose team: [https://www.compose.io/articles/loss-prevention-with-rethinkdb/](https://www.compose.io/articles/loss-prevention-with-rethinkdb/)

It details how you can properly take advantage of a multi node cluster but to do this at the application level you need to pass options to `r.tableCreate`. Currently this is inaccessible from Thinky and would be extremely useful to expose.

##### Proposed Solution

1. Support a `table: {...}` key that is passed to `thinky.createModel` and thus `Model.new` and the `Model` constructor.
2. Store these options on the model just like the `_pk` is already stored.
3. Pass these options to `tableCreate` on line 205 of `model.js`

I'm happy to open a PR if you are okay with this solution.

##### Current Solution

Until this is supported, I am currently doing something like the following to get this to work:

```javascript
'use strict';

const thinky = require('thinky');
const config = require('app/config');

exports = module.exports = thinky(config.rethinkdb);

// Patch r.tableCreate
let tableCreate = exports.r.tableCreate;
exports.r.tableCreate = function(name, options) {
  options.replicas = config.rethinkdb.replicas || 1;
  return tableCreate.call(exports.r, name, options);
};
```

**Comment by neumino** [comment]
Make sense, want to send a PR ? :)

**Comment by AndrewBarba** [comment]
Opened here https://github.com/neumino/thinky/pull/490

**Comment by neumino** [comment]
Awesome, released as 2.3.1.
Docs were updated too. Thanks!
